.envrc

```diff
@@ -1 +1,3 @@
-use nix
+if command -v nix &>/dev/null; then
+    use flake
+fi
```

Comment:
I would think direnv already does the check when setting `use flake` ?

Reply:
When I last tested it multiple months ago, direnv would fail if it didn't have the flake module loaded and I omitted the `command` check. It's definitely possible that has changed in recent versions of direnv, though; feel free to send a link to a changelog entry or commit and I'll gladly simplify this.